core/src/scss/utilities/mixins/flex/_flex-grid-push.scss

```diff
@@ -1,15 +1,14 @@
 @charset "UTF-8";
 
-//
-// @flex-grid-push($size: 1, $columns: $su-grid-columns)
-//
-// Pushes a grid column by changing its left margin.
-//
-// $size - integer: The number of columns to 'push'
-// $columns - integer: The number of columns in the grid.
-//
-// Style guide: Mixins.Flex.flex-grid-push
-//
+///
+/// Pushes a grid column by changing its left margin.
+///
+/// @name flex-grid-push
```

Comment:
I recall that we found out the flex push mixins aren't working as intended. Keep it for now or remove?

Reply:
Keep it documented. Perhaps someone will submit a PR to fix.